<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>store_modify2</class>
 <widget class="QWidget" name="store_modify2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>:/qt 图标/修改图标/write.ico</activeon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 12pt &quot;楷体&quot;;
background-color: rgb(144, 144, 144);
color:rgb(0, 0, 0)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="6" column="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="cursor">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>现有库存：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="1" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 11pt &quot;微软雅黑&quot;;
color:rgb(255, 255, 255);
background-color: rgb(83, 83, 83);
border-radius: 8px;
border: 3px groove rgb(98, 98, 98);
border-style: outset;
}
QPushButton:pressed{
border-style: inset;
}
QPushButton:hover{
background-color:rgb(0, 0, 0);
}

</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 11pt &quot;微软雅黑&quot;;
color:rgb(255, 255, 255);
background-color: rgb(83, 83, 83);
border-radius: 8px;
border: 3px groove rgb(98, 98, 98);
border-style: outset;
}
QPushButton:pressed{
border-style: inset;
}
QPushButton:hover{
background-color:rgb(0, 0, 0);
}

</string>
       </property>
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 75 11pt &quot;微软雅黑&quot;;
color:rgb(255, 255, 255);
background-color: rgb(83, 83, 83);
border-radius: 8px;
border: 3px groove rgb(98, 98, 98);
border-style: outset;
}
QPushButton:pressed{
border-style: inset;
}
QPushButton:hover{
background-color:rgb(0, 0, 0);
}

</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>管 理 员：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>仓库名称：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>仓库编号：</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>备    注：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>仓库类型：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>库存容量：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>状   态：</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>请选择管理员</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="2" colspan="4">
    <widget class="QLineEdit" name="lineEdit_9"/>
   </item>
   <item row="7" column="4">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>联系电话：</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="cursor">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
